src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py

```diff
@@ -959,6 +989,26 @@ def body(self):
             JMESPathCheck('resourceGroup', resource_group)
         ])
 
+class VMSSDiagnosticsInstallTest(VCRTestBase):
```

Comment:
same as above

---
In reply to: [76078544](https://github.com/Azure/azure-cli/pull/737#discussion_r76078544) [](ancestors = 76078544)